sql/core/src/main/scala/org/apache/spark/sql/execution/command/AlterTableAlterViewCommandParser.scala

```diff
@@ -29,22 +29,26 @@ import org.apache.spark.sql.types.StructType
 
 
 /**
- * Helper object to parse alter table commands.
+ * Helper object to parse alter table or alter view commands.
  */
-object AlterTableCommandParser {
+object AlterTableAlterViewCommandParser {
```

Comment:
actually, I believe this will be removed in the near future. All changes you make here should be in SparkSqlParser.scala instead.